openassessment/templates/openassessmentblock/staff_area/oa_staff_grade_learners.html

```diff
@@ -1,19 +1,19 @@
 {% load i18n %}
 
-<div class="staff__grade__control ui-toggle-visibility is--collapsed">
```

Comment:
Note to reviewers: no more usage of "ui-toggle-visibility" in the staff grading tools (we will purge this from LMS portion in a later story). Instead we are using JQuery slideUp/slideDown which properly marks collapsed content as display none (while still supporting animation).